_includes/manuals/nightly/1.2_release_notes.md

```diff
@@ -4,6 +4,11 @@ This section will be updated prior to the next release.
 
 ### Headline features
 
+#### Array, Boolean and other types support in Parameters
+
+Global parameters now support the same types as smart variables and smart class parameters.
+As a part of this change search by parameters has been modified so that instead of parameter value, searching based on parameter name is allowed. ([#4127](https://projects.theforeman.org/issues/4127))
```

Comment:
I think searching by name was already allowed so it's probably clearer to state that searching by parameter value is no longer allowed.